inference-engine/src/mkldnn_plugin/nodes/mkldnn_roll_node.cpp

```diff
@@ -170,7 +159,7 @@ void MKLDNNRollNode::rollImpl() {
     const size_t elementSize = sizeof(DataType);
 
     const size_t nIterations = totalElements / blockSize;
-    const auto strides = dataEdge->getMemory().GetDescWithType<CpuBlockedMemoryDesc>().getStrides();
+    const auto strides = dataEdge->getMemory().GetDescWithType<BlockedMemoryDesc>()->getStrides();
```

Comment:
```suggestion
    const auto& strides = dataEdge->getMemory().GetDescWithType<BlockedMemoryDesc>()->getStrides();
```

Reply:
`GetDescWithType` return pointer to copy desc

Reply:
Yep, agree, my bad.